Source/Core/RectangleGeometry.js

```diff
@@ -802,6 +802,7 @@ define([
         options.textureMatrix = textureMatrix;
         options.tangentRotationMatrix = tangentRotationMatrix;
         options.size = options.width * options.height;
+        options.stRotation = stRotation;
```

Comment:
Is this actually used anywhere?  I traced through the code visually and didn't see this particular options object including `stRotation`.